Tests/MintTests/MintTests.swift

```diff
@@ -121,7 +121,7 @@ class MintTests: XCTestCase {
 
         try mint.bootstrap()
 
-        let package = PackageReference(repo: "yonaskolb/SimplePackage", version: "4.0.0")
+        let package = PackageReference(repo: "yonaskolb/SimplePackage", version: PackageReference.Revision.branch(name: "4.0.0"))
```

Comment:
Once Revision has `ExpressibleByStringLiteral` conformance, these can all stay the same